src/server/api.ts

```diff
@@ -15,6 +15,7 @@ import cors from '@koa/cors';
 import { InitializeGame } from '../core/initialize';
 import * as StorageAPI from './db/base';
 import { Server, Game } from '../types';
+import { update } from '../../dist/types/src/core/action-creators';
```

Comment:
Is this used somewhere? This is breaking the CI tests because you’re trying to import from `dist` in `src`. In any case, I don’t think this is being used, so it’s probably a case of just deleting this line.

Reply:
Oh, I probably pressed tab accidentally while writing the update function and typescript auto imported this. Definitely not using it, removed it, thanks.